framework/axSGreaseSkeleton.js

```diff
@@ -149,5 +201,9 @@ const DYNAMIC_TWEAK_ATTRIBS = [];
 const DYNAMIC_TWEAKS = [
 ];
 
+/** add your specific initialization here, so that if you ever update the framework from new skeleton your inits are not overridden. */
```

Comment:
```suggestion
/** Add your specific initialization here, so that if you ever update the framework from new skeleton your inits are not overridden. */
```